index.js

```diff
@@ -0,0 +1,38 @@
+var express = require('express');
+var mongoose = require('mongoose');
+var path = require('path');
+var spaRoute = require('./utils/spa-server-route');
+var truckRouter = require('./routes/propRoutes');
+var foodTypeRouter = require('./routes/foodTypeRoutes');
+var truckAPIRouter = require('./routes/propRoutes');
+
+var messenger = require('./custom_messenger');
+//call like function
+messenger('message from Gandalf');
+//messenger.message('message from bilbo');
+
+
+var db = mongoose.connect('mongodb://localhost/propsAPI');
+var app = express();
+
+
+app.use(express.static('public'));
+//app.use(bodyParser.urlencoded({ extended: false }));
+
+app.use('/trucks', truckRouter);
+app.use('/food-types', foodTypeRouter);
+app.use('/food-trucks', foodTypeRouter);
+app.use('/today', foodTypeRouter);
+app.use('/props', foodTypeRouter);
+
+app.use(express.static(root));
```

Comment:
If you're going to use root as a variable here, then you have to define it first.